### DISCUSSION TITLE
Limit Feedin Power

### DISCUSSION THREAD

**Discussion by Jannis87** [wills106/homeassistant-solax-modbus#428] [Q&A]
Hi There,

Recently I got a PV system installed on the roof. It is a QCells system (SolaX G4). The first thing I did was to install a Modbus adapter to have all the data in Home Assistant. For this I used this wonderful extension.
Now the extension offers many options that I do not understand. Is it possible to set the input power to 0?
I live in Germany and strictly speaking I need a new meter to operate the system. Without feed-in, however, that would be fine.

Translated with www.DeepL.com/Translator (free version)

**Comment by infradom** [comment]
Not sure what you mean by feedin. If you mean export to grid, then you can set export_control_user_limit = 0 to avoid export to grid
